build_daemon/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: build_daemon
-version: 0.6.0
+version: 0.7.0
```

Comment:
We have some more breaking changes coming so it would be good to roll these all together in one release (which will be the 1.0.0 release)

Reply:
SGTM. Not too worried about how this gets released. Please just point me in the right direction :)